components/resources/library/src/commonTest/kotlin/org/jetbrains/compose/resources/TestResourceReader.kt

```diff
@@ -6,15 +6,14 @@ internal class TestResourceReader : ResourceReader {
 
     override suspend fun read(path: String): ByteArray {
         readPathsList.add(path)
-        return DefaultResourceReader.read(path)
+        return getDefaultResourceReader().read(path)
     }
 
     override suspend fun readPart(path: String, offset: Long, size: Long): ByteArray {
         readPathsList.add("$path/$offset-$size")
-        return DefaultResourceReader.readPart(path, offset, size)
+        return getDefaultResourceReader().readPart(path, offset, size)
     }
 
-    override fun getUri(path: String): String {
-        return DefaultResourceReader.getUri(path)
-    }
+    override fun getUri(path: String): String =
+        getDefaultResourceReader().getUri(path)
```

Comment:
Please, don't change a code style

Reply:
if you don't agree with the current, it is OK. but should be discussed and applied by separate commit for the whole file